src/driver/test_point_aggregator.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test Point Aggregator.
+
+Given a collection of true reach values and a corresponding collection
+of reach estimates, computes a single row DataFrame with summary statistics
+on the differences between the true values and the estimated values.
+"""
+
+import numpy as np
+import pandas as pd
+from typing import List
+
+from wfa_planning_evaluation_framework.models.reach_point import ReachPoint
+
+
+# A list of functions for computing aggregation metrics.  Each function
+# takes as input the true reach and modeled reach points and produces
+# as output a scalar value (could be float, int or string).
+AGGREGATORS = {
```

Comment:
Looks like the most compact way of defining a class of methods. Good to learn.

Reply:
I like this table-driven style because it is very flexible and powerful, although in this case the benefits of using it are not that great.